workspace-hack/Cargo.toml

```diff
@@ -25,7 +25,8 @@ bit-set = { version = "0.5.3" }
 bit-vec = { version = "0.6.3" }
 bitflags-dff4ba8e3ae991db = { package = "bitflags", version = "1.3.2" }
 bitflags-f595c2ba2a3f28df = { package = "bitflags", version = "2.5.0", default-features = false, features = ["serde", "std"] }
-bstr = { version = "1.9.1" }
+bstr-6f8ce4dd05d13bba = { package = "bstr", version = "0.2.17" }
+bstr-dff4ba8e3ae991db = { package = "bstr", version = "1.9.1" }
```

Comment:
is this intentional or do we want a single bstr?

Reply:
hah good question -- this happened because I pulled in the `similar` library to do diffs, which depends on bstr 0.2. I filed https://github.com/mitsuhiko/similar/issues/66 about this.